sdk-actors/src/main/java/io/dapr/actors/client/ActorClient.java

```diff
@@ -59,7 +59,7 @@ public ActorClient(ResiliencyOptions resiliencyOptions) {
    * @param overrideProperties Override properties.
    */
   public ActorClient(Properties overrideProperties) {
-    this(buildManagedChannel(overrideProperties), null);
+    this(buildManagedChannel(overrideProperties), null, overrideProperties.getValue(Properties.API_TOKEN));
```

Comment:
I also needed to fix how properties override work because it was not handling it correctly for Dapr API Token. Now, it works too and all ITs require sidecar calls to have dapr-api-token header.